### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by chipco**

### CODE REVIEW

File: RapidHouseNumbers.js [hunk 1]
```diff
@@ -324,7 +325,7 @@
         if (rapidHNNext) {
           rapidHNtoolbarButton = undefined;
 
-          addHouseNumber = rapidHNNext.previousSibling; // recursiveSearchFor(mutation.addedNodes, ['add-house-number']);
+          addHouseNumber = rapidHNNext.previousElementSibling; // recursiveSearchFor(mutation.addedNodes, ['add-house-number']);
```
Comment (SaiCode-DEV) [inline RapidHouseNumbers.js]: Hi! Whats your Intention behind that change?
`console.log("siblings", rapidHNNext.previousSibling, rapidHNNext.previousElementSibling);`
results for me as a null for previousElementSibling.

**Comment by SaiCode-DEV** [comment]
Hi! Whats your Intention behind that change?
`console.log("siblings", rapidHNNext.previousSibling, rapidHNNext.previousElementSibling);`
results for me as a null for previousElementSibling.

**Comment by chipco** [comment]
Hi! There is some whitespace between the elements and `previousSibling` picks up a "\n"  instead of the desired input. `previousElementSibling` ignores whitespace.

**Comment by chipco** [comment]
It's to fix this issue here: https://greasyfork.org/en/scripts/35931-wme-rapid-house-numbers/discussions/252293#comment-518932

**Comment by SaiCode-DEV** [comment]
messed around a bit more and still cant get to previousElementSibling. What Browser are you using?

**Comment by chipco** [comment]
I'm on Chrome with Tampermonkey.

You are right about the null result. This helps the script move over the if condition.
previousSibling returns some whitespace and crashes the code when trying to access classList of this non element.

image

**Comment by chipco** [comment]
I did some more testing and pushed a new commit.

There were other issues preventing the save of the next value when closing RHN.

For me `window.getComputedStyle(node)` always returned an object with all attributes as empty string. So I replaced that.

All the `ItemDisabled` stuff is deprecated, the WME no longer corresponds with the commented use case.

File: RapidHouseNumbers.js [hunk 2]
```diff
@@ -478,8 +461,8 @@
           className => !node.classList.contains(className),
         ) === -1
       ) {
-        const { display } = window.getComputedStyle(node);
-        const visible = display !== "none";
+        const display = node.attributeStyleMap.get("display");
```
Comment (SaiCode-DEV) [inline RapidHouseNumbers.js]: attributeStyleMap doesnt work in firefox!
grafik

**Review by SaiCode-DEV — COMMENTED** [review]
attributeStyleMap is not working in firefox. Please check both browsers.

File: RapidHouseNumbers.js (hunk 2 above)
Comment (chipco) [reply RapidHouseNumbers.js]: I adapted the visibility check to work on both Chrome and Firefox

File: RapidHouseNumbers.js (hunk 1 above)
Comment (chipco) [reply RapidHouseNumbers.js]: previousSibling returns some whitespace and crashes the code when trying to access classList of this non element

**Review by mapomatic — APPROVED** [review]

**Pull request merged**